### ISSUE TITLE
Open sourcing the official website?

### ISSUE THREAD

**Issue by adirh3** [blogifierdotnet/Blogifier#308]
**Is your feature request related to a problem? Please describe.**
Blogifier is amazing, I can get high quality blog website up within minutes, but I believe for many scenarios it will be shipped with a small website.

**Describe the solution you'd like**
The official Blogifier website gets most of the common things done, like landing page, documentations with MD support, features, about and support pages.
If possible, it would be awesome if the Blogifier official website was opened sourced, as it completely matches the look and feel of Blogifier.

**Describe alternatives you've considered**
Provide a small customizable website based on the official website, with option to add documentations in editor just like blog posts, add features, etc.

Thanks!

**Comment by david-randoll** [comment]
I agree with this. I was browsing the site and found nothing on how to install the blog. Would be nice for the community to create an installation guide for the different operating systems. I created an installation guide and I think it could help others as I had to figure out on my own how to host a Dotnet 6 app on Ubuntu. So I made a blog post on how to do it and would like to add it to the Blogifier website if people are finding it helpful.

https://blog.davidrandoll.com/posts/host-blogifier-for-free-on-aws
